src/components/Button/Button.tsx

```diff
@@ -10,13 +15,15 @@ export type ButtonVariant =
   | "text"
   | "danger"
   | "secondaryDanger";
+export type ButtonSubtype = "default" | "info" | "success" | "error" | "warning";
```

Comment:
Kudos; Thanks for adding this. Good improvement to the component interface.

Reply:
thanks Simon! :)